src/Forms/ElementalAreaField.php

```diff
@@ -208,4 +211,48 @@ public function performReadonlyTransformation()
             ->setName($this->getName())
             ->addExtraClass('elemental-area--read-only');
     }
+
+    public function setSubmittedValue($value, $data = null)
+    {
+        // Content comes through as a JSON encoded list through a hidden field.
+        $this->setValue(Convert::json2array($value));
```

Comment:
It'd be good to return `$this` for chaining, and also these methods could use doc blocks